2021Q1/translation_pt_BR.po

```diff
@@ -521,71 +578,71 @@ msgstr "Insira um comentário."
 
 msgctxt "{\"cxt\": \"question_option_text\", \"id\": 4051178332}"
 msgid "Older than 4.0"
-msgstr ""
+msgstr "Anterior à 4.0"
 
 msgctxt "{\"cxt\": \"question_option_text\", \"id\": 4051178333}"
 msgid "4.0"
-msgstr ""
+msgstr "4.0"
 
 msgctxt "{\"cxt\": \"question_option_text\", \"id\": 4051178334}"
 msgid "4.1"
-msgstr ""
+msgstr "4.1"
 
 msgctxt "{\"cxt\": \"question_option_text\", \"id\": 4051178335}"
 msgid "4.2"
-msgstr ""
+msgstr "4.2"
 
 msgctxt "{\"cxt\": \"question_option_text\", \"id\": 4051178336}"
 msgid "4.3"
-msgstr ""
+msgstr "4.3"
 
 msgctxt "{\"cxt\": \"question_option_text\", \"id\": 4051178337}"
 msgid "4.4"
-msgstr ""
+msgstr "4.4"
 
 msgctxt "{\"cxt\": \"question_option_text\", \"id\": 4051178338}"
 msgid "4.5"
-msgstr ""
+msgstr "4.5"
 
 msgctxt "{\"cxt\": \"question_option_text\", \"id\": 4051178339}"
 msgid "4.6"
-msgstr ""
+msgstr "4.6"
 
 msgctxt "{\"cxt\": \"question_option_text\", \"id\": 4051178340}"
 msgid "4.7"
-msgstr ""
+msgstr "4.7"
 
 msgctxt "{\"cxt\": \"question_option_text\", \"id\": 4051178341}"
 msgid "4.8"
-msgstr ""
+msgstr "4.8"
 
 msgctxt "{\"cxt\": \"question_option_text\", \"id\": 4086918770}"
 msgid "5.0"
-msgstr ""
+msgstr "5.0"
 
 msgctxt "{\"cxt\": \"question_option_text\", \"id\": 4051178342}"
 msgid "Nightly \"build\""
-msgstr ""
+msgstr "Build Noturno"
```

Comment:
Since `Nightly build` is a technical term and we don't have a good translation for `Build`, I think that is better to don't translate this term.
```suggestion
msgstr "<i>Nightly \"build\</i>""
```

Reply:
Poxa, Build Noturno é o canal dos nossos amigos MPV's ;p